client/gutenberg/extensions/contact-form/index.js

```diff
@@ -42,18 +43,43 @@ export const settings = {
 			type: 'string',
 			default: '',
 		},
-		submit_button_text: {
+		submitButtonText: {
 			type: 'string',
 			default: __( 'Submit' ),
 		},
-		has_form_settings_set: {
+		customBackgroundButtonColor: { type: 'string' },
+		customTextButtonColor: { type: 'string' },
```

Comment:
Can we get the color value from the DOM here?

Reply:
Wouldn't it be too messy?

Reply:
Gutenberg is able and willing to serialize attributes to and parse attributes from the DOM, it has built in capabilities.

Here's an example: https://github.com/Automattic/wp-calypso/blob/1113c84911876df72d58e5e1e6db8d87032acd6a/client/gutenberg/extensions/tiled-gallery/index.js#L39-L50

Reply:
Has anyone looked at this?

We may be able to skip this convoluted section entirely: https://github.com/Automattic/wp-calypso/pull/29722/files#diff-cc53dae3c0199e53a36e0f3b9de311e7R25

We'd need to get and parse the color classname from the serialized block.

Reply:
We are using fallback colors in a similar way that core uses them in the button component:

https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/edit.js#L31